### DISCUSSION TITLE
documentation for connecting to external services contains very little info

### DISCUSSION THREAD

**Discussion by kirk86** [nicolargo/glances#3226] [General]
Hi and thanks for glances.
I was wondering if there is any better documentation in regards to connecting to external services like grafana, influxdb etc.?

I've read the manual but doesn't provide a comprehensive example on how to set things up.
Let me demonstrate with a simple example:

We have server X running glances and we want to connect the stats to grafana.
Personally I have a grafana instance running live on https://username.grafana.net,
This instance of grafana is not installed on server X.
Now there are a couple hiccups IMHO.
First, in the docs it mentions that there's a dedicated grafana dashboard and it links into `.json` file.
Question how do you load that in the live grafana service that I have https://username.grafana.net?
The only sources there require to connect some sort db presumable running on server X.
If possible I am trying to avoid installing unnecessary databases only to have a dashboard of stats, is that possible?
Second, if I execute `glances --export influxdb` what does that do? There's no output message, nothing?
Possibly I can only infer that it dumps some file with stats somewhere, but it would be nice to get an informative message on what operations where executed and where are the outputs stored?

Thanks!

**Comment by nicolargo** [comment]
You can have a look on this post: https://www.tecmint.com/install-glances-influxdb-grafana-to-monitor-centos-7/
